src/Orc.FilterBuilder/Runtime/Serialization/FilterSchemeSerializerModifier.cs

```diff
@@ -8,23 +8,27 @@ public class FilterSchemeSerializerModifier : SerializerModifierBase<FilterSchem
 {
     public override void SerializeMember(ISerializationContext context, MemberValue memberValue)
     {
-        if (string.Equals(memberValue.Name, nameof(FilterScheme.TargetType)))
+        if (!string.Equals(memberValue.Name, nameof(FilterScheme.TargetType)))
```

Comment:
this will make code harder to refactor in the future in case multiple members should be supported.

Reply:
ok